gradle/validation/dependency-analyze.gradle

```diff
@@ -24,7 +24,7 @@ allprojects { prj ->
 
     analyzeClassesDependencies {
       warnUsedUndeclared = false // means fail build if UsedUndeclared found
-      warnUnusedDeclared = true // means only log warning if UnusedDeclared found
+      warnUnusedDeclared = false // means fail build if UnusedDeclared found
```

Comment:
Now that all the warnings are fixed - this will fail the build if new ones are introduced.